sources/packages/web/src/constants/award-constants.ts

```diff
@@ -55,22 +55,22 @@ export const AWARDS: AwardDetail[] = [
   },
   {
     awardType: PartTimeAwardTypes.CSLP,
-    description: "Canada Student Loan for Part-time Studies",
+    description: "Canada Student Loan for Part-Time Students",
```

Comment:
Part-time with lower case "t".

Reply:
I know the AC has upper-case but all the system uses lower case. Please check.

Reply:
Nice catch. It's this one place where "time" is capitalized. It's now updated.